File: AlarmManagerDemo/app/src/main/java/rohksin/com/alarmmanagerdemo/SecondActivity.java
package rohksin.com.alarmmanagerdemo;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;

/**
 * Created by Illuminati on 11/6/2017.
 */

public class SecondActivity extends AppCompatActivity {

    private TextView textView ;

    @Override
    public void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.second_layout);
        textView = (TextView)findViewById(R.id.textView);

        String text = getIntent().getStringExtra("SomeValue");
        textView.setText("From AlarmManager "+text);

    }



}

File: Kotlin_Basic_Example/app/src/main/java/com/asu/kotlin_basic_example/SecondActivity.kt
package com.asu.kotlin_basic_example

import android.os.Bundle
import android.widget.Toast
import com.google.android.material.snackbar.Snackbar
import androidx.appcompat.app.AppCompatActivity

import kotlinx.android.synthetic.main.activity_second.*

class SecondActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)
        textView.setOnClickListener{
            Toast.makeText(this, "This works", Toast.LENGTH_LONG).show();
        }

    }

}
